apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -200,43 +173,7 @@ async function processPageFellowships(page) {
   blocks.push({
     slug: "events",
     title: "Events",
-    items: Array.from({ length: 30 }, (_, i) => ({
-      id: i,
-      title: "Event title going on two or even three lines",
-      category: "Topic Name",
-      date: "2023-02-11",
-      excerpt: [
-        {
-          children: [
-            {
-              text: "Lorem ipsum dolor sit amet consectetur adipiscing elit tempus nibh cursus, urna porta sagittis non eget taciti nunc sed felis dui, praesent ullamcorper facilisi euismod ut in platea laoreet integer. Lorem ipsum dolor sit amet consectetur ",
-            },
-          ],
-        },
-      ],
-      image: {
-        id: "63d2622aafe25f6469605eae",
-        alt: `Grant ${i}`,
-        prefix: "media",
-        filename: "Rectangle 113.jpg",
-        mimeType: "image/jpg",
-        filesize: 257010,
-        width: 1236,
-        height: 696,
-        createdAt: "2023-01-26T11:21:14.868Z",
-        updatedAt: "2023-01-26T11:21:14.868Z",
-        url: "/images/featured-event.svg",
-      },
-      link: {
-        href: `/events/${i}`,
-      },
-      registerLink: {
-        href: `/register/events/${i}`,
-      },
-      registerText: "Register ",
-      status: ["upcoming", "past"][Math.floor(Math.random() * 2)],
-      featured: i === 0,
-    })),
+    items: events,
     config: {
```

Comment:
@kilemensi how should we handle this config on CMS?.

Reply:
Global @koechkevin. From the [docs](https://payloadcms.com/docs/configuration/globals):

> Globals are perfect for things like header nav, site-wide banner alerts, **`app-wide localized strings`**, and other "global" data that your site or app might rely on.
>